packages/react-components/react-dialog/src/components/DialogTitle/renderDialogTitle.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import type { DialogTitleState, DialogTitleSlots } from './DialogTitle.types';
+
+/**
+ * Render the final JSX of DialogTitle
+ */
+export const renderDialogTitle_unstable = (state: DialogTitleState) => {
+  const { slots, slotProps } = getSlots<DialogTitleSlots>(state);
+
+  return (
+    <>
+      <slots.root {...slotProps.root}>{slotProps.root.children}</slots.root>
+      {slots.closeButton && (
```

Comment:
should the close button not be rendered by default ?

Reply:
No, only if is a `non-modal` dialog according to design spec. 🤷🏼